pkg/patterns/declarative/reconciler.go

```diff
@@ -189,6 +191,25 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 				log.Error(err, "failed to reconcile status")
 			}
 		}
+
+		status, err := utils.GetCommonStatus(instance)
+		if err != nil {
+			log.Error(err, "failed to get status")
+		}
+		compiledStatus := utils.CompileErrors(status)
+		fmt.Println("compiling errors")
```

Comment:
Should be `log`, rather than using fmt which will go to stdout.

Reply:
Sure. I will changing it